### ISSUE TITLE
Refresh Client List after finishing ClientDetailsFragment

### ISSUE THREAD

**Issue by therajanmaurya** [openMF/android-client#380]
**Summary:** 

When we open any Client Details and delete the client profile image and come back to client list.
client profile image is deleted successfully but the image is visible because client list is not refreshed, the image coming from the cache.

Is this good to refresh client list?

**Comment by mayank-kgp** [comment]
@therajanmaurya Is there any way to remove cache from Glide? 
If not, there can be only option as you suggested. 
What do you think?

**Comment by Pritom14** [comment]
Yes, it is a good practice to refresh the client list after deletion. Normally , I use `notifyDataSetChanged()`.
Any suggestions?

**Comment by AkshGautam** [comment]
@therajanmaurya do we need to add it ? If yes,I would like to work on this.

**Comment by tarun0** [comment]
@AkshGautam Yes, you can work on it. But implement it using some approach that won't refresh the whole client list again because that will be a redundant network call. 

Hint: when the request to delete the client image is sent, you must be receiving some response as to whether it was successful or not. Try thinking about that and make use of callbacks. 

If you have some other approach, do share it as well for discussion prior to working on it.

**Comment by AkshGautam** [comment]
@tarun0 we can look for the status code which the sender receives after request if the operation was successful or not. If the Status_ok signal is received we could make the changes. I would have to look into what status code is retrieved. is this right?

**Comment by tarun0** [comment]
Yes, something like that only; i.e. if the response for the request to delete client image is successful (you'll have to check what kind of JSON response it sends), update the image for that particular client. The emphasis should be on avoiding the network call.
